src/main/resources/com/ncc/aif/aida_ontology.shacl

```diff
@@ -1067,6 +1067,9 @@ aida:ClusterShape
    # may bear system-private data
    sh:property aida:PrivateDataPropertyShape ;
 
+   # my provide a link
+   sh:property aida:LinkPropertyShape ;
```

Comment:
AIDA-701 states 

> linkTarget must not be an empty string, but no further validation on the value is required.

I wonder if the `aida:LinkPropertyShape` should be enhanced to include this check.  I've started a Teams conversation on the topic, so we'll see.  If we do end up checking for an empty linkTarget, then we should probably have an InvalidTest that creates an empty link.

Reply:
Okay, based on offline conversation, it looks like nothing should be added to AIF other than the link itself, but as the NIST restriction is finalized, perhaps we'll add a check for empty to restricted AIF.